releases/release-1.29/release-notes/maps/pr-119142-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 119142
+releasenote:
+  text: |-
+    `kube-apiserver`: adds `--authentication-config` flag for reading `AuthenticationConfiguration`
+    files. `--authentication-config` flag is mutually exclusive with the existing `--oidc-*`
+    flags.
```

Please surround this into single quotes to avoid a probable issue as in [this PR#2096](https://github.com/kubernetes/sig-release/pull/2096/files#diff-f9f63b5952664a20155ad75e12cbccc6d8e0304dc644c2333cdc8c05acfb0f3fR4).

```suggestion
    '`kube-apiserver`: adds `--authentication-config` flag for reading `AuthenticationConfiguration`
    files. `--authentication-config` flag is mutually exclusive with the existing `--oidc-*`
    flags.'
```

---

Missed that one, I usually use `|-`

---

This issue happens when a paragraph starts with a backtick (`) character.

---

Yes, but that's exactly where I was told to use `|-` actually... which is why I changed all comments that start with a backtick to that format, since that change was (in my mind) what solved that issue. This was the approach used during the 1.28 cycle, using `|-` when the line starts with a backtick.

If it isn't, I will update the shadows on this,  but it was done on purpose.

---

@fsmunoz  Oh, I see. It's excellent that setting `|-` solves the problem. Sorry for all the wrong suggestions, I'll remove all those related to that.